features/shared-storage.yml

```
@@ -1,7 +1,7 @@
-draft_date: 2024-09-09
-name: Shared Storage API
-description: TODO
+name: Shared storage
+description: The `sharedStorage` API stores data across sites. Unlike `localStorage`, third-parties can write to shared storage, which you can read from the origin browsing context.
```

This one is tricky. It's quite a complex API. Devs sort of need to understand privacy implications of ads before being able to use it. I think the description could do a bit of better job at making the main use case explicit. The use of "site", "third-parties", "origin", and "browsing context" assume knowledge here. Let me try something:

```suggestion
description: The `sharedStorage` API stores data to a shared space where the data can then be processed without the ability to track users across the different sites they visit. A common use case is measuring the reach of third-party ads without using user-tracking cookies."
```